config/exchanges.json

```diff
@@ -1415,32 +1396,12 @@
     },
     "rotation": 0
   },
-  "NL->NO": {
-    "capacity": [
-      -700,
-      700
-    ],
-    "lonlat": [
-      5.795449,
-      55.859727
-    ],
-    "parsers": {
-      "exchange": "statnett.fetch_exchange"
-    },
-    "rotation": 0
-  },
   "NL->NO-NO2": {
     "parsers": {
       "exchangeForecast": "ENTSOE.fetch_exchange_forecast",
       "exchange": "statnett.fetch_exchange"
     }
   },
-  "NO->SE": {
-    "parsers": {
-      "exchange": "statnett.fetch_exchange",
-      "exchangeForecast": "ENTSOE.fetch_exchange_forecast"
```

Comment:
@maxbellec we still want to keep the exchange forecast here. Maybe ENTSOE has forecasts on BZ level?